allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -82,15 +117,43 @@ def tokens_to_indices(self, tokens: List[Token], vocabulary: Vocabulary) -> Inde
                     "Using PretrainedTransformerIndexer but field text_id is not set"
                     f" for the following token: {token.text}"
                 )
-
-        # The mask has 1 for real tokens and 0 for padding tokens. Only real tokens are attended to.
-        mask = [1] * len(indices)
-
-        return {"token_ids": indices, "mask": mask}
+        return indices
+
+    def _postprocess_output(self, output: IndexedTokenList) -> IndexedTokenList:
```

Comment:
Please add a docstring here. Even though this is a protected method it is much easier to understand with documentation.